ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -44,9 +46,15 @@ azure_connection::list_objects(std::vector<std::string> &objects) const
     return 0;
 }
 
+/* 
+ * Puts an object (blob) given a reference to the filename
```

Comment:
Let's remove (blob) here.

Reply:
resolved in https://github.com/wiredtiger/wiredtiger/pull/8718/commits/89a53338336237e74307020901a82f9cc1620e53